File: linear-data-structure/DinamicArray.java
import java.util.*;
public class DinamicArray {
  public static void main (String args[]) {
    var v = new ArrayList<Integer>();
    v.add(1);
    v.add(2);
    v.add(3);
    System.out.println(v.get(2));
    v = new ArrayList<>(Arrays.asList(
          new Integer[]{1,2,3,4,5,6,7}
          ));
    v.set(0, v.get(6)); //acceso
    int idx = v.indexOf(5); //búsqueda
    v.remove(idx); //eliminación
    Collections.sort(v);
    for(var e:v) 
      System.out.print(e + " ");
    System.out.println();

  }
}

File: linear-data-structure/DinamicArray.cpp
#include <bits/stdc++.h> 
using namespace std;

int main () {
  vector<int> v; //sin elementos iniciales
  v.push_back(1);
  v.push_back(2);
  v.push_back(3);
  printf("%d\n", v[2]);
  v = {1,2,3,4,5,6,7};
  v[0] = v[6]; //acceso
  auto it = find(v.begin(), v.end(), 5); //búsqueda
  v.erase(it); //eliminación
  sort(v.begin(), v.end());
  for(auto& e: v) {
    printf("%d ", e);
  }
  printf("\n");
  return 0;
}



